### PR TITLE
Only issue a MouseMove event if the scroll offset actually changed.

### LINKED ISSUE #8098: Tooltips don't work if there is a visible scrollpanel
This affects the lobby (faction and spawn tooltips) and #8065.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/13347234-tooltips-don-t-work-if-there-is-a-visible-scrollpanel?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by pchote**
Fixes #8098.  Easiest testcase is the lobby.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs [hunk 1]
```diff
@@ -74,11 +74,15 @@ protected void SetListOffset(float value, bool smooth)
 			targetListOffset = value;
 			if (!smooth)
 			{
+				var oldListOffset = currentListOffset;
 				currentListOffset = value;
 
 				// Update mouseover
-				var mi = new MouseInput(MouseInputEvent.Move, MouseButton.None, 0, Viewport.LastMousePos, Modifiers.None, 0);
-				Ui.HandleInput(mi);
+				if (oldListOffset != currentListOffset)
```
Comment (penev92) [inline OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs]: "Comparison of floating point numbers with equality operator. Possible loss of precision while rounding values."
Do we care about that here?

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs]: I would say we don't care about that in something so minute and simple.

**Comment by penev92** [comment]
:+1:

**Comment by obrakmann** [comment]
Confirmed the fix. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/5baf8e51ebd9a9b9b891e7cc8a8f67bd0f62b0fa%5E...5baf8e51ebd9a9b9b891e7cc8a8f67bd0f62b0fa)

**Pull request merged**